fluentui_controls/src/main/java/com/microsoft/fluentui/tokenized/controls/FileCard.kt

```diff
@@ -0,0 +1,197 @@
+                        if (onClick != null) Modifier.clickable(
+                            interactionSource = interactionSource,
+                            indication = rememberRipple(),
+                            enabled = true,
+                            onClick = onClick,
+                            role = Role.Button
+                        ) else Modifier
+                    ),
+                horizontalAlignment = Alignment.CenterHorizontally,
+            ) {
+                if (previewImageDrawable != null) {
+                    Image(
+                        painterResource(id = previewImageDrawable), contentDescription = null
+                    )
+                } else if (previewImageVector != null) {
+                    Image(
+                        imageVector = previewImageVector,
+                        contentDescription = null
+                    )
+                }
+                Row(
+                    modifier = Modifier.padding(
+                        textContainerPadding
+                    ),
+                    verticalAlignment = Alignment.CenterVertically
+                ) {
+                    Icon(
+                        modifier = Modifier.size(iconSize),
+                        icon = FluentIcon(textIcon),
+                        tint = iconColor
+                    )
+                    Spacer(modifier = Modifier.width(iconTextSpacing))
+                    Column {
+                        BasicText(
+                            text = text,
+                            style = textTypography.merge(TextStyle(color = textColor))
+                        )
+                        Spacer(
+                            modifier = Modifier
+                                .fillMaxWidth()
+                                .height(textSubTextSpacing)
+                        )
+                        BasicText(
+                            text = subText,
+                            style = subTextTypography.merge(TextStyle(color = subTextColor))
+                        )
+                    }
+                }
+            }
+            if (isPreviewAvailable && actionOverflowIcon != null) {
+                Box(
+                    modifier = Modifier
+                        .align(Alignment.TopEnd)
+                        .padding(actionOverflowPadding)
+                        .clip(RoundedCornerShape(actionOverflowCornerRadius))
+                        .background(actionOverFlowBackgroundColor)
+                        .clickable(
+                            interactionSource = MutableInteractionSource(),
+                            indication = rememberRipple(),
+                            enabled = true,
+                            onClick = actionOverflowIcon.onClick ?: {},
+                            role = Role.Button
+                        )
+                        .semantics(true, properties = {
+                            role = Role.Button
```

Comment:
Why is the required? Role is alsready mentioned in clickable

Reply:
removed